contents/handbook/engineering/support-hero.md

```diff
@@ -209,6 +209,24 @@ Ask in [#team-website-and-docs](https://posthog.slack.com/archives/C01V9AT7DK4)
 
 _Note: The PostHog.com community currently uses a separate authentication system from PostHog Cloud. There are [plans](https://github.com/PostHog/posthog.com/issues/5847) to support other types of authentication so a visitor doesn't have to create a separate account for asking questions._
 
+## How do I handle a bug report or feature request?
+
+For feature requests from low priority users, [give them this link](https://github.com/PostHog/posthog/issues/new/choose) and suggest they open a feature request.
+
+For bug reports from normal and high priority users (assuming you've confirmed it's a bug, and that there's not already an open bug report):
+
+1. [Open a bug report](https://github.com/PostHog/posthog/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) on our GitHub repo
+2. Be sure to include a link to the insight (or other), below the repo steps
+3. Include "From: `https://URL_for_ZenDesk_ticket`" in the `additional info` section of the bug comment (where the URL is for the ZenDesk ticket where the customer reported the bug)
+4. Reply to the user to thank* them for alerting us to the bug. Let them know you've opened a bug report and provide a link to it.
+5. Let them know they can follow the bug report on GitHub for updates.
+6. When sending the reply, change the ticket from `Open` to `Pending`
```

Comment:
Why do we suggest to keep these ticket open in Zendesk? We should close them, since they are resolved from the support's side.

Reply:
@MarconLP  Step 6 suggests changing "from `Open` to `Pending`, because they usually reply to say "thanks" or with questions, which changes it back to `Open`, so we can `Solve` it after that exchange.